### DISCUSSION TITLE
How Do I Setup A Shop To Buy Claim Blocks For Players

### DISCUSSION THREAD

**Discussion by Noobymen** [GriefPrevention/GriefPrevention#1972] [Q&A]
Can someone tell me what i need to do and what other plugin i need to setup a admin shop that can sell claim blocks for players to buy with the money they make from there jobs

**Comment by bigpresh** [comment]
You would need Vault installed, and a Vault-compatible economy plugin - then see the economy integration section of the docs on how to enable purchasing claim blocks:
https://docs.griefprevention.com/configuration/#economy-integration
